dev/scraped-articles/apnews_articles_2024-08-16_14-22-20.csv

```diff
@@ -0,0 +1,50 @@
+title,description,url,section,level,image_url,scraped_at
```

Comment:
we _probably_ don't want to store these article csv's in github. Is there a compelling reason to keep these files or are these just temporary?